Adafruit_DAP_STM32.cpp

```diff
@@ -274,8 +274,11 @@ bool Adafruit_DAP_STM32::verifyFlash(uint32_t addr, const uint8_t *data,
   uint8_t buf[4 * 1024];
 
   while (size) {
-    uint32_t const count = min(size, sizeof(buf));
-
+    #if defined(ESP8266) || defined(ESP32)
```

Comment:
please cast any non-matching types so we can continue to use min() rather than having a special ifdef

Reply:
Hi @ladyada,
unfortunately, my current knowledge only goes so far.... so I might need a pointer (no pun)
Losing the ifdef would clean things up a bit.

Do you meen:  uint32_t bytes = min((int)count, CHUNK_SIZE); ?

Reply:
@Onno-Dirkzwager try to see if this work without the ifdef. Basically you want both arg has the same type 

```C
      uint32_t const count = min(size, (uint32_t) sizeof(buf));
```

Reply:
Thank you @hathach, my English failed me!